modules/tinymce/src/core/main/ts/textpatterns/core/BlockPattern.ts

```diff
@@ -55,10 +55,15 @@ const applyPattern = (editor: Editor, match: BlockPatternMatch): boolean => {
   return true;
 };
 
+const sortPatterns = <P extends Pattern>(patterns: P[]): P[] => Arr.sort(patterns, (a, b) => {
+return b.start.length - a.start.length
+});
```

Comment:
```suggestion
const sortPatterns = <P extends Pattern>(patterns: P[]): P[] =>
  Arr.sort(patterns, (a, b) => b.start.length - a.start.length);
```
I think this is more readable?